src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -9869,15 +9874,13 @@ public static Vector128<T> Invoke(Vector128<T> x)
             {
                 if (typeof(T) == typeof(float))
                 {
-                    if (Sse.IsSupported)
-                    {
-                        return Sse.Reciprocal(x.AsSingle()).As<float, T>();
-                    }
+                    if (Sse.IsSupported) return Sse.Reciprocal(x.AsSingle()).As<float, T>();
+                    if (AdvSimd.IsSupported) return AdvSimd.ReciprocalEstimate(x.AsSingle()).As<float, T>();
```

Comment:
nit: This is inconsistent with how `FusedMultiplyAdd` is handled above (it is `Isa.IsSupported` with nested `type checks`; while this is `type checks` with nested `Isa.IsSupported`).

It might be nice to make them consistent (I personally prefer `Isa.IsSupported` on top, but it's not a strong preference)